<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>testirregularwidgets</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>center</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QWidget" name="widget_10" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="widget_11" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QQtWidget" name="widget_12" native="true"/>
           </item>
           <item row="1" column="2">
            <widget class="QWidget" name="widget_13" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QWidget" name="widget_14" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string> center mask</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QWidget" name="widget_5" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QWidget" name="widget_3" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QQtIrregularWidget" name="widget" native="true"/>
           </item>
           <item row="1" column="2">
            <widget class="QWidget" name="widget_6" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QWidget" name="widget_4" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>zoom mask</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QQtIrregularWidget" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>zoom keepaspectratio</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QQtIrregularWidget" name="widget_21" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>tiled mask</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QQtIrregularWidget" name="widget_7" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Other mask</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QQtIrregularWidget" name="widget_15" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="QQtIrregularWidget" name="widget_16" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QQtIrregularWidget" name="widget_17" native="true"/>
        </item>
        <item row="1" column="1">
         <widget class="QQtIrregularWidget" name="widget_18" native="true"/>
        </item>
        <item row="2" column="0">
         <widget class="QQtIrregularWidget" name="widget_19" native="true"/>
        </item>
        <item row="2" column="1">
         <widget class="QQtIrregularWidget" name="widget_20" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>button mask</string>
       </attribute>
       <widget class="QQtIrregularPushbutton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QQtIrregularWidget" name="widget_22" native="true">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QQtIrregularWidget" name="widget_23" native="true">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QQtIrregularWidget" name="widget_24" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>91</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QQtIrregularWidget</class>
   <extends>QWidget</extends>
   <header>qqtirregularwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQtWidget</class>
   <extends>QWidget</extends>
   <header>qqtwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQtIrregularPushbutton</class>
   <extends>QPushButton</extends>
   <header>qqtirregularpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
